boards/shields/mcp2515/adafruit_can_picowbell.overlay

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 Joseph Yates
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pico_spi {
+	status = "okay";
+	cs-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>;
+
+	mcp2515_adafruit_can_picowbell: can@0 {
+		compatible = "microchip,mcp2515";
+		spi-max-frequency = <1000000>;
+		int-gpios = <&gpio0 21 GPIO_ACTIVE_LOW>;
```

Comment:
For arduino header there would be something like `int-gpios = <&arduino_header 12 (GPIO_ACTIVE_LOW)>;`. You probably need to introduce pico_header. @erwango what is your opinion?

Reply:
Added "pico_header" GPIO nexus node & updated shield ".overlay" to use the "pico_header" now :)